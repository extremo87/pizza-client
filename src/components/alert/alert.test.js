import React from 'react';
import Alert from './alert';
import renderer from 'react-test-renderer';
import configureStore from 'redux-mock-store';
import NameSpace from "../../reducers/name-space.js";

const mockStore = configureStore([]);

const store = mockStore({
  [NameSpace.SERVICE]: {
    error: null
  }
});

it(`Alert conponent renders correctly`, () => {
  const tree = renderer
    .create(
        <Alert store={store}
          onClearError={() => {}}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
