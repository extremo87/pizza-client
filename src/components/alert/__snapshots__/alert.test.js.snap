// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert conponent renders correctly 1`] = `null`;
